.github/workflows/lint-charts-pr.yml

```diff
@@ -0,0 +1,39 @@
+name: Lint Charts PR
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.12.1
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+          check-latest: true
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.6.0
```

Comment:
Have a look at latest changelog [v2.6.1](https://github.com/helm/chart-testing-action/releases/tag/v2.6.1). They upgraded ct version. If helpful, make changes accordingly.

Reply:
bumped to v2.6.1